static_src/actions/user_actions.js

```diff
@@ -233,30 +234,50 @@ const userActions = {
   },
 
   createUserAndAssociate(userGuid) {
-    const orgGuid = OrgStore.currentOrgGuid;
+    let entityGuid;
+    const entityType = UserStore.currentlyViewedType;
+
+    if (entityType === 'org_users') {
```

Comment:
there's a constant somewhere that has `'org_users'`, refer to that. (it may need to be moved.